Mathlib/Geometry/Manifold/ContMDiff/Defs.lean

```diff
@@ -479,6 +479,23 @@ theorem contMDiffOn_iff :
     convert hdiff x (f x) (extChartAt I x x) (by simp only [hx, mfld_simps]) using 1
     mfld_set_tac
 
+/-- zero-smoothness on a set is equivalent to continuity on this set. -/
+theorem contMDiffOn_zero_iff :
+    ContMDiffOn I I' 0 f s ↔ ContinuousOn f s := by
+  rw [contMDiffOn_iff]
+  refine ⟨fun h ↦ h.1, fun h ↦ ⟨h, ?_⟩⟩
+  intro x y
+  rw [show ((0 : ℕ∞) : WithTop ℕ∞) = 0 by rfl, contDiffOn_zero]
+  apply ContinuousOn.comp (continuousOn_extChartAt _)
+  · apply ContinuousOn.comp h
```

Comment:
Same here. (You could even do `apply h.comp ((continuousOn_extChartAt_symm _).mono inter_subset_left)`, right?)